lib/release/release.js

```diff
@@ -86,13 +90,15 @@ class StencilRelease {
      * @param gitData.ahead
      * @returns {boolean}
      */
-    checkGitData(gitData) {
+    checkGitData(gitData, options) {
         if (gitData.remotes.length === 0) {
             throw new Error('No git remote repository found');
         }
 
-        if (gitData.current !== 'master') {
-            throw new Error('Not on master branch, please checkout master branch to proceed');
+        if (gitData.current !== options.branch) {
+            throw new Error(
+                `Not on ${options.branch} branch, please checkout master branch to proceed`,
```

Comment:
```suggestion
                `Not on ${options.branch} branch, please checkout ${options.branch} branch to proceed`,
```
I assume it is possible situation when gitData is on branch `test1` and our input is `test2`. In this case error message will be confusing.